README.md

```diff
@@ -17,17 +17,71 @@ Qualle provides a command-line interface to train
 and evaluate models. In addition, a REST webservice for predicting
 the recall of a MLC result is provided.
 
-### Command line interface (CLI)
-In order to run the CLI, you must install the packages with [poetry](https://python-poetry.org).
-The interface is then accessible from the module ``qualle.main``. To
-see the help message, run (inside the Qualle directory)
+## Installation
 
-``python -m qualle.main -h``
+### Requirements
 
+Python 3.8 or 3.9 is required.
 
-### Train
-In order to train a model you have to provide a training data file.
-This file has to be a tabular-separated file (tsv) in the format (tabular is marked with ``\t``)
+### With pip
```

Comment:
Evt. vor dieser Zeile doch nochmal einen Satz "You can choose one of three installation methods:"?

Reply:
Vielleicht lieber: "Choose one of these installation methods:". Ich mag abschließende Listen nicht so gerne, da es sicherlich noch andere Installationsmethoden gibt.

Reply:
ok